profitbricks-rest/src/test/java/org/apache/jclouds/profitbricks/rest/features/VolumeApiLiveTest.java

```diff
@@ -53,13 +55,19 @@ public void teardownTest() {
    @Test
    public void testCreateVolume() {
       assertNotNull(dataCenter);
+      
+      HashSet<String> sshKeys = new HashSet<String>();
+      sshKeys.add("hQGOEJeFL91EG3+l9TtRbWNjzhDVHeLuL3NWee6bekA=");
 
       testVolume = volumeApi().createVolume(
               Volume.Request.creatingBuilder()
               .dataCenterId(dataCenter.id())
               .name("jclouds-volume")
-              .size(3)
+              .size(4)
               .licenceType(LicenceType.LINUX)
+              .type(VolumeType.HDD)
+              .image("362c903a-df61-11e5-80a4-52540005ab80")
```

Comment:
I'm afraid of introducing more complexity to the unit test. If we add this this test wouldn't be atomic any more. 
These tests won't be ran too often so it wouldn't hurt to change them every time we run them.